polylogue/doctor.py

```
@@ -4,7 +4,7 @@
 import shutil
 from dataclasses import dataclass
 from pathlib import Path
-from typing import Dict, List, Optional, Tuple
+from typing import Any, Dict, List, Optional, Tuple
```

Ruff (0.15.9)

[warning] `typing.Dict` is deprecated, use `dict` instead

(UP035)